lib/bolt/cli.rb

```diff
@@ -559,6 +550,41 @@ def initialize_project
       ok ? 0 : 1
     end
 
+    def migrate_project
+      if inventory.version == 2
+        ok = true
+      else
+        inventory_file = inventory_source
```

Comment:
Inventory source can return the constant `Bolt::Inventory::ENVIRONMENT_VAR` which points to the `BOLT_INVENTORY` environment variable. That allows you to specify inventory data (not a path to an inventory file). So I think we would want to either skip the migrate, or perhaps load the inventory data, check for v2, and print the conversion?

Reply:
I think printing the conversion there makes sense - I can see users having docker workflows with inventory exported in a non-inventory file, and having a way to convert and copy-paste seems valuable. I think it'd also be fine to skip migration in that case for now and ticket it - we may want to define that behavior a bit more as a team.

Reply:
Removed the `inventory_source` method and do error handling in place. I agree with Lucy to skip migration for the environment var for now and ticket it separately.